source/index.txt

```diff
@@ -10,6 +12,105 @@ MongoDB Shell
    :depth: 1
    :class: singlecol
 
+.. include:: /includes/admonitions/fact-mdb-shell-beta.rst
+
+The |mdb-shell| is an interactive JavaScript shell interface to MongoDB,
+which provides a powerful interface for users to test queries and
+operations directly with the database. The |mdb-shell| also provides a
+fully functional JavaScript environment for use with MongoDB.
+
+The |mdb-shell| is available as a standalone package in the MongoDB
+download center.
+
+Download and Install the |mdb-shell|
+------------------------------------
+
+.. tabs-platforms::
+
+   .. tab::
+      :tabid: windows
+
+      Hello, I am Windows content.
+
+   .. tab::
+      :tabid: macOS
+
+      Hello, I am macOS content.
+
+   .. tab::
+      :tabid: linux
+      
+      Hello, I am Linux (rpm / deb) content.
+
+Multi-Line Operations in the |mdb-shell|
+----------------------------------------
+
+Run the ``.editor`` command within the |mdb-shell| to open the editor
+and manage comprehensive multiline functions:
+
+.. example::
+
+   Run the following command within the |mdb-shell|:
+
+   .. code-block:: javascript
+
+      .editor
+
+   The |mdb-shell| enters editor mode:
+
+   .. code-block:: javascript
+      :copyable: false
+
+      // Entering editor mode (^D to finish, ^C to cancel)
+      const { Writable } = require('stream');
+
+        const myWritable = new Writable({
+          write(chunk, encoding, callback) {
+            // ...
+          },
+          writev(chunks, callback) {
+            // ...
+          }
+        });
+
+- Press ``Cmd/Ctrl`` + ``d`` to finish and run your function.
+
+- Press ``Cmd/Ctrl`` + ``c`` to cancel and exit without running your
+  function.
+
+When you close the editor, the |mdb-shell| does not save any functions
+you had entered into the editor.
+
+.. note::
+
+   Currently, ``editor()`` is the only way to paste multiline commands
+   into the |mdb-shell|.
+
+
+Comparison of the |mdb-shell| and Legacy ``mongo`` Shell
```

Comment:
[suggestion] "|mdb-shell| vs. Legacy ``mongo`` Shell" is more immediately clear to me as a section topic. Up to you, I don't know if the style guide says anything about using/not using "vs."

Reply:
Tweaked this title.